pkg/db/driver_mysql.go

```diff
@@ -3,47 +3,59 @@ package db
 import (
 	"database/sql"
 	"fmt"
-	"net/url"
+	"net"
 	"strconv"
 
-	_ "github.com/go-sql-driver/mysql"
+	"github.com/go-sql-driver/mysql"
 	"github.com/golang-migrate/migrate/v4/database"
-	"github.com/golang-migrate/migrate/v4/database/mysql"
+	migrateMysql "github.com/golang-migrate/migrate/v4/database/mysql"
+	"github.com/justtrackio/gosoline/pkg/log"
 )
 
 const DriverMysql = "mysql"
 
 func init() {
-	connectionFactories[DriverMysql] = NewMysqlDriverFactory()
+	connectionFactories[DriverMysql] = NewMysqlDriver
 }
 
-func NewMysqlDriverFactory() DriverFactory {
-	return &mysqlDriverFactory{}
-}
+func NewMysqlDriver(logger log.Logger) (Driver, error) {
+	log := &mysqlLogger{logger: logger}
+	mysql.SetLogger(log)
```

Comment:
In driver_cratedb.go you are calling this func still with Factory suffix. Should we have this unified across all drivers?

Reply:
dropped the factory suffix